bigbluebutton-html5/imports/ui/components/polling/polling-graphql/component.tsx

```diff
@@ -0,0 +1,401 @@
+import React, {
+  ElementRef, useEffect, useMemo, useRef, useState,
+} from 'react';
+import { useMutation, useSubscription } from '@apollo/client';
+import { defineMessages, useIntl } from 'react-intl';
+import { Meteor } from 'meteor/meteor';
+import AudioService from '/imports/ui/components/audio/service';
+import Checkbox from '/imports/ui/components/common/checkbox/component';
+import PollService from '/imports/ui/components/poll/service';
```

Comment:
Prefer implements it on own service to isolate the component or move it on global scope